docs/index.html

```diff
@@ -681,8 +683,7 @@ <h3 slot="header">Badges</h3>
         <il-footer>
 
             <div slot="contact" class="il-footer-contact uofi_address">
-                <p><a href="https://vigilant-fishstick-2k4pppq.pages.github.io/">Application Development UX Standards</a></p>
-            </div>
+<p><a href="https://appdev.illinois.edu/">Application Development Standards</a></p>            </div>
```

Comment:
@merrifld Should this be `Components` instead of `Standards`? 

```suggestion
<p><a href="https://appdev.illinois.edu/">Application Development Components</a></p>            </div>
```

Reply:
I ask since I know we've made a similar change in several places.

Reply:
I checked and the link text matches the page title, so I think this is correct, as is.